app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -73,6 +74,7 @@ export class DashboardItemPage extends Component {
     const { formatMessage } = this.props.intl;
     return (
       <PageLayout title={formatMessage(messages.pageTitle)}>
+        <AddDashboardButton />
```

Comment:
I think it's better to create a new component for title and pass this component to a `title` prop. It will help to avoid absolute positioning and also will be useful for adding breadcrumbs

Reply:
Will be implemented in scope of EPMRPP-35139